### PR TITLE
Create Greeter for First Launch

### PR DESCRIPTION
**Pull request by ayushyamitabh**
Some package requirement changes, app name fixes.

Created the Greeter for the first launch of the app, currently this only contains the UI implementation. It still has to be plugged into the server side so it can download the model from our server during the first run.

Stil need to implement the logic for only showing this window on the first run.

Let me know if there should be any additional screens. 
I will work on implementing a basic server next, that way we can starting connecting everything.

Screen Shot 2019-10-23 at 11 45 16 AM
Screen Shot 2019-10-23 at 11 45 39 AM
Screen Shot 2019-10-23 at 11 45 57 AM
Screen Shot 2019-10-23 at 11 46 03 AM

### CODE REVIEW

File: app/package.json [hunk 1]
```diff
@@ -31,7 +39,7 @@
 	"build": {
 		"appId": "com.your-domain",
```
Comment (cpanican) [inline app/package.json:40]: Should we change the appId too?

**Review by cpanican — APPROVED** [review]
LGTM 👍

**Review by gerryzqx — CHANGES_REQUESTED** [review]
Looks pretty good to me. One thing I think would make it more readable, would be to make the fonts for the description of each screen bigger.

**Comment by ayushyamitabh** [comment]
@gerryzqx @cpanican Squash and merge if all good

Fixed - you can try to run to see other screens, this also makes sure the app runs on windows
image
image

**Pull request merged**